### DISCUSSION TITLE
Layout changed?

### DISCUSSION THREAD

**Discussion by Dirge70** [enfyna/fx-new-tab#2] [General]
Hello, just today my icon layout has changed. I used to have all my icons in 2 rows,, but now they are lined up in 3. They also all have a white border around them. Is there a setting to lock it to 2 rows, and disable the white borders?

Edit: Sorry to keep talking to myself, but the number of icon rows change with the width of the window. Before it would keep the layout no matter what the window width was. Now it jumps between 2 and 3 rows.

**Comment by enfyna** [comment]
Hi! This is really interesting actually I removed all the responsivity classes so it should always be the same row whatever the window width. And I dont remember changing the shortcut borders do you mean these : 

![huh](https://github.com/enfyna/fx-new-tab/assets/91965312/6dd30ab9-6896-45dd-82e1-636372c16b66)

Can you send me your settings so I can look at it ? You can export your settings from bottom of the settings page and before uploading here you can delete the shortcuts and notes keys from the json file for privacy concerns.

**Reply by Dirge70** [reply]
Yes, that white border. I can change the shape of them (box, rounded box, circle), but can not just remove them.
![white_border](https://github.com/enfyna/fx-new-tab/assets/104225510/8ac56144-0708-4523-849b-847da2551556)

Here is the json file:
[settings.json](https://github.com/user-attachments/files/16086881/settings.json)

**Reply by enfyna** [reply]
Ah ok I read the settings thanks for sending it. The reason for the 2-3 row jump is that I changed how the shorcut sizing is applied so you need to refresh the value. In shortcut settings change the shortcut size and revert it. It should save the new value. I dont add auto value updaters to my code because I feel like that is unnecessary bloat and it is not reasonable to check for an update everytime a new tab opens. 
As for the border there is no option for it currently but it shouldnt be hard to implement i guess i can work on this tonight or tomorrow.

**Reply by Dirge70** [reply]
Thanks! That fixed the issue. I do have another feature request about the minimum icon size. I'll add it to the issues. (I should have put this there, sorry!)

**Reply by enfyna** [reply]
No problem! Could you add an issue for removing shortcut border option as well ?

**Reply by Dirge70** [reply]
Done. Thank you for making this plugin!
